specification/resources/resource-manager/Microsoft.Authorization/stable/2023-01-01/privateLinks.json

```diff
@@ -0,0 +1,409 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a private link association, associate scope to rmpl.": {
+            "$ref": "./examples/PutPrivateLinkAssociation.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateLinkAssociation"
+        ],
+        "operationId": "PrivateLinkAssociation_Delete",
+        "description": "Delete a PrivateLinkAssociation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkAssociationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": ""
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a private link association.": {
+            "$ref": "./examples/DeletePrivateLinkAssociation.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PrivateLinkAssociation"
+        ],
+        "operationId": "PrivateLinkAssociation_Get",
+        "description": "Get a PrivateLinkAssociation(resource-level).",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkAssociationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Returns information about the private link association.",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkAssociation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a private link association, associate scope to rmpl.": {
+            "$ref": "./examples/GetPrivateLinkAssociation.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/providers/Microsoft.Authorization/privateLinkAssociations": {
+      "get": {
+        "tags": [
+          "PrivateLinkAssociation"
+        ],
+        "operationId": "PrivateLinkAssociation_List",
```

Comment:
Last time we had to do a fix to existing API to add support for List in our SDK. If we look at this in Previous version 2020-05-01 this is GET changing this to List operation here and adding a GET (Single resource get operation above). DO you see any concerns?

Reply:
This LIST call looks correct to me. the above operation would be the single resource GET